wfe2/wfe.go

```diff
@@ -35,8 +35,8 @@ import (
 // measured_http.
 const (
 	directoryPath  = "/directory"
-	newRegPath     = "/acme/new-reg"
-	regPath        = "/acme/reg/"
+	newAcctPath    = "/acme/new-acct"
```

Comment:
In the current draft this is referred to as `acme/new-account`

Reply:
but ¯\_(ツ)_/¯

Reply:
@rolandshoemaker I think the spec only standardizes "new-account" as the directory entry. I believe the actual value is up to implementations to decide. I'm happy to use "/acme/new-account" for the path value but it's an implementation choice (As I understand it) not a spec mandate.

Reply:
Yeahhh, the [Directory](https://ietf-wg-acme.github.io/acme/draft-ietf-acme-acme.html#directory) section just provide an example directory which uses `new-account`. That said since client devs are likely to use the spec as a reference and it probably makes sense to make things as similar as possible unless we have a good reason for it.

Reply:
> That said since client devs are likely to use the spec as a reference it probably makes sense to make things as similar as possible unless we have a good reason for it.

Hmm. I actually interpret it the other way. Since client devs have likely been cheating and using hard coded endpoint URLs instead of dereferencing the `/directory` as intended by the spec we should explicitly choose a new value here. [We did similar in Pebble](https://github.com/letsencrypt/pebble/blob/master/wfe/wfe.go#L32:L43).

@jsha Do you have a preference?

Reply:
Yeah, I agree it's both fine, and actually somewhat good, to have a different name and force directory use here. Makes it clear that the example URLs in the spec are just examples.